### ISSUE TITLE
Docker image

### ISSUE THREAD

**Issue by kubitre** [horgh/videostreamer#13]
Can u dockerize this app?

**Comment by kubitre** [comment]
@horgh

**Comment by horgh** [comment]
I added a Dockerfile. Let me know if you see issues with it!

**Comment by kubitre** [comment]
it turned out to be a very large image(~500mb). Maybe look in the base image will be of scratch or maybe alpine with a fully static binary?

**Comment by kubitre** [comment]
I tried to build a static binary, though not yet successfully.

**Comment by horgh** [comment]
Right, [distroless's](https://github.com/GoogleContainerTools/distroless) would probably be ideal. The problem as you probably found is the ffmpeg libraries. If one can figure out the libraries to copy into the runtime image then it should work, but I'm not sure an easy way to do that. I'd probably accept a PR!
